code/game/objects/items/ropechainbola.dm

```diff
@@ -208,7 +208,7 @@
 	name = "noose"
 	desc = "Abandon all hope."
 	icon = 'icons/roguetown/misc/tallstructure.dmi'
-	pixel_y = 10
+	pixel_z = 10
```

Comment:
Not even TG does this and they actually have a good reason to and this conflicts every change in my base pixel pr :(((